packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_p2wpkh.test.ts

```diff
@@ -40,14 +40,7 @@ beforeEach(async () => {
   await providers.randomizeEllipticPair()
   await container.waitForWalletBalanceGTE(101)
 
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 1.1) // 1.1
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 5.5) // 6.6
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 10.566) // 17.166
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 15.51345) // 32.67945
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 20) // 52.67945
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 37.98) // 90.65945
-  await fundEllipticPair(container, providers.elliptic.ellipticPair, 9.34055) // 100
```

Comment:
I guess same for this too

Reply:
https://github.com/JellyfishSDK/jellyfish/pull/1362/files/cda88e8d27e78e89e2101733d1953457fc2d54af#r874739378